name/templates/name/about.html

```diff
@@ -90,31 +88,39 @@
         },
```

Comment:
after last items in the JSON, I don't think there should be a comma (in the last two Search API examples). Also we could get rid of the space after this one.